bin/cli.js

```diff
@@ -124,30 +124,41 @@ function invoke(env) {
       `-x [${filetypes.join('|')}]`,
       'Specify the knexfile extension (default js)'
     )
-    .action(() => {
-      const type = (argv.x || 'js').toLowerCase();
-      if (filetypes.indexOf(type) === -1) {
-        exit(`Invalid filetype specified: ${type}`);
-      }
-      if (env.configuration) {
-        exit(`Error: ${env.knexfile} already exists`);
+    .action(async () => {
+      try {
+        const packageDirectory = await pkgDir(process.cwd());
+        const consumingPackageJson = require(path.resolve(
+          packageDirectory,
+          'package.json'
+        ));
+        const consumingPackageIsModule = consumingPackageJson.type === 'module';
```

Comment:
This logic appears to be duplicated in at least 3 places.  Perhaps consolidate it into a reusable function?

(On a related note: I suspect that `Liftoff` probably makes this information available already.  If so, then it would probably be best to use the values that `Liftoff` provides.  Otherwise, we'll probably encounter some weird discrepancies when `knex` / `Liftoff` disagree about which `package.json` should be used)